### ISSUE TITLE
Gateway doesn't allow OPTIONS request

### ISSUE THREAD

**Issue by yatharthranjan** [RADAR-CNS/RADAR-Docker#73]
We get a 401 unauthorised when making a pre-flight options request. Right now, I have added a conditional block in nginx config on the production server to handle this. Its something like 
```nginx
          if ($request_method = OPTIONS){
                  add_header Access-Control-Allow-Origin "*";
                  add_header Access-Control-Allow-Methods "GET, POST, HEAD, OPTIONS";
                  add_header Access-Control-Allow-Headers "Authorization, Content-Type";
                  add_header Access-Control-Allow-Credentials "true";
                  add_header Content-Length 0;
                  add_header Content-Type text/plain;
                  return 200;
          }
```
Should we do this in the Nginx or should the gateway handle this CORS issue?

**Comment by blootsvoets** [comment]
Fixing this in nginx would be easier from my POV. Btw I think the response code should be `204`.
See: https://gist.github.com/Stanback/7145487
with the suggestion:
> If you have multiple server blocks that you want to use CORS with, you can also save this as a separate file in /etc/nginx/cors and within your server blocks, enter "include cors;" which will include the file.
This would make it easy to enable CORS for selected locations.

**Comment by yatharthranjan** [comment]
Closed by #74
